res/css/views/rooms/_RoomHeader.scss

```diff
@@ -154,6 +154,10 @@ limitations under the License.
     max-height: 2.4em;
 }
 
+.mx_RoomHeader_topic .mx_Emoji {
+    font-size: inherit;
```

Comment:
I think we can afford 1.2x sized emoji, assuming they don't throw the room header out of alignment

Reply:
Unfortunately, the 1.2 factor led to the vertical cropping I had mentioned [here](https://github.com/matrix-org/matrix-react-sdk/pull/8215#discussion_r842025310). Using `inherit` also ensures that emoji in HTML topics have the same size as in plain text topics (where they are not wrapped in `.mx_Emoji`).

Reply:
can we get that documented in-code? 😇

Reply:
Ah, good idea, yes!